### PR TITLE
fix: set fixed versions requirements except for jina encoders part 3

### PR DESCRIPTION
**Pull request by JoanFM**

### CODE REVIEW

File: encoders/numeric/FastICAEncoder/requirements.txt [hunk 1]
```diff
@@ -1 +1 @@
-sklearn
+scikit-learn==0.24
```
Comment (florian-hoenicke) [inline encoders/numeric/FastICAEncoder/requirements.txt:1]: should be changed to `scikit-learn==0.24.0`

Comment (JoanFM) [reply encoders/numeric/FastICAEncoder/requirements.txt:1]: why? when doing pip show, it shows 0.24

File: encoders/video/VideoPaddleEncoder/requirements.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
-paddlepaddle
-paddlehub
-Pillow
-six>=1.14.0
-protobuf>=3.11.0
+paddlepaddle==1.8.5
+paddlehub==1.8.3
+Pillow==8.0.1
```
Comment (florian-hoenicke) [inline encoders/video/VideoPaddleEncoder/requirements.txt:3]: Pillow==8.1.0

**Review by florian-hoenicke — CHANGES_REQUESTED** [review]

File: encoders/video/VideoPaddleEncoder/requirements.txt (hunk 2 above)
Comment (JoanFM) [reply encoders/video/VideoPaddleEncoder/requirements.txt:3]: is it an improvement or a requirement? If tests pass we can take in separate PRs to upgrade dependencies, but taking for each executor the newest dependencies is overkilling now

File: encoders/numeric/FastICAEncoder/requirements.txt (hunk 1 above)
Comment (florian-hoenicke) [reply encoders/numeric/FastICAEncoder/requirements.txt:1]: https://github.com/scikit-learn/scikit-learn/releases/tag/0.24.0
Maybe it works as well but my pycharm is going crazy when using 0.24

File: encoders/video/VideoPaddleEncoder/requirements.txt (hunk 2 above)
Comment (JoanFM) [reply encoders/video/VideoPaddleEncoder/requirements.txt:3]: Anyone is welcome later to upgrade dependencies, but out of the scope of this PR

Comment (florian-hoenicke) [reply encoders/video/VideoPaddleEncoder/requirements.txt:3]: okay

**Review by florian-hoenicke — APPROVED** [review]

**Review by florian-hoenicke — APPROVED** [review]

**Pull request merged**